Examples/Run/Reconstruction/Common/RecInput.cpp

```diff
@@ -7,9 +7,9 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 #include "ActsExamples/Detector/IBaseDetector.hpp"
-#ifdef ACTS_PLUGIN_ONNX
-#include "Acts/Plugins/Onnx/MLTrackClassifier.hpp"
-#endif
+//#ifdef ACTS_PLUGIN_ONNX
+//#include "Acts/Plugins/Onnx/MLTrackClassifier.hpp"
+//#endif
```

Comment:
Just seeing this, does that plugin still work? Is this change breaking it?

Reply:
Oh, I missed that one ... @XiaocongAi  can clarify I hope

Reply:
That plugin still works. But I don't think we need those lines here. It still works.

Reply:
Can we remove them then ?

Reply:
Oops! The whole `RecInput` files should be removed actually now. Fixed.